packages/forms/src/Components/BaseFileUpload.php

```diff
@@ -137,6 +145,13 @@ public function disk($name): static
         return $this;
     }
 
+    public function keepFilename(bool | Closure $keepFilename = true): static
```

Comment:
Instead of "keep filename", maybe "preserve filename" more eloquently describes this feature?

Reply:
I was also undecided with keepOriginalFilename.
Confirmed preserveFilename?

Reply:
Yeah - `preserve` is synonymous to `keepOriginal`.